buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/gradlebuild/unittestandcompile/UnitTestAndCompilePlugin.kt

```diff
@@ -250,8 +252,12 @@ class UnitTestAndCompilePlugin : Plugin<Project> {
             configureJvmForTest()
             addOsAsInputs()
 
-            doFirst {
-                if (BuildEnvironment.isCiServer) {
+            if (BuildEnvironment.isCiServer) {
+                retry {
```

Comment:
We need to make sure that retries are not configured for the performance test task. Or at least that the plugin doesn't try to retry failed tests there.

Reply:
Done.